src/dependencies/token/IERC20Flexible.sol

```diff
@@ -0,0 +1,11 @@
+pragma solidity 0.6.4;
+
+/// @title Flexible ERC20 interface
+/// @author Melon Council DAO <security@meloncoucil.io>
+/// @notice support standard ERC20 and BadERC20
+/// @notice A 'BadERC20Token' token is one that uses an old version of the ERC20 standard. Basically, this version does not return anything from `transfer` and `transferFrom`, whereas most modern implementions of ERC20 return a boolean to indicate success or failure.
```

Comment:
@luongnt95 We realized recently that we should not have multiple of the same tag (e.g., `@dev` or `@notice`). So this could be:

```
/// @notice Here is the first comment.
/// Here is the second comment.